packages/Heading/src/Heading.js

```diff
@@ -49,29 +49,14 @@ const defaultProps = {
 const safeValue = n => (n < 1 || Number.isNaN(n) ? 6 : Math.min(n, 6));
 
 function getElementProps(safeDisplayLevel, safeLevel, props) {
-  const {
-    a11yText,
-    children,
-    displayLevel,
-    domRef,
-    hasDivider,
```

Comment:
Need this `hasDivider` prop on the actual component for styling purposes.  In fact, we can rely on `moreProps` to propagate most of these props to the component.

Reply:
I think it starts to get confusing when we peel away certain props from `props` when there are this many only to return another object.... it starts to get confusing as to what is actually being used or not, as I see `level` isn't .... so it shouldn't need to be deconstructed in this function

I would prefer all props to be deconstructed in one place than in separate functions personally. (Ie just on line 69)